types/FormatHelpers.d.ts

```diff
@@ -56,4 +57,18 @@ export interface FormatHelpers {
   ) => (token: TransformedToken) => string;
   fileHeader: (args: FileHeaderArgs) => string;
   formattedVariables: (args: FormattedVariablesArgs) => string;
-}
\ No newline at end of file
+  minifyDictionary: (dictionary: object) => object;
+  getTypeScriptType: (value: any) => string;
+  iconsWithPrefix: (
+    prefix: string,
+    allTokens: DesignToken[],
+    options: object
+  ) => string;
+  sortByReference: (dictionary: Dictionary) => any;
```

Comment:
Could we type this a bit stronger? `sortByReference` returns a sorting function:

```suggestion
  sortByReference: (dictionary: Dictionary) => (a: DesignToken, b: DesignToken) => number;
```